ios/ios_testing_references.md

```diff
@@ -1,5 +1,15 @@
 # iOS Testing References
 
+## Why do we write tests?
+
+Project stakeholders are not always sold on the benefits of writing tests. Here are a few arguments to help convince them:
+
+* Automated tests allow you to catch bugs as they are being introduced, instead of being caught by users
+* You can integrate new code faster and with more confidence if there are tests looking for regression bugs
```

Comment:
I would rephrase this around the idea that it will catch regressions. This might confuse people thinking they need to write specific tests.